R/aaa.R

```diff
@@ -13,3 +13,11 @@ new_function <- function(args = NULL,
   x[[length(x) + 1L]] <- value
   x
 }
+
+topNamespaceName <- function(env = parent.frame()) {
+  env <- topenv(env)
+  if (isNamespace(env))
+    getNamespaceName(env)
+  else
+    NULL
```

Comment:
```suggestion
  if (isNamespace(env)) {
    getNamespaceName(env)
  } else {
    NULL
  }
```

Reply:
well,  `else NULL`  already being a no-op,  I do not really prefer the extra `{ .. }`.
My personal style uses a 1-liner for something so simple:
```
  if (isNamespace(env)) getNamespaceName(env)  # else NULL
```
.. applying  _KISS_

Reply:
I just remembered/rediscovered why I had originally written `else NULL`—it was to make a `NULL` return value print; otherwise, the implicit `else` is returned `invisible()`. 

One could wrap with `(if(...))`, but I had opted for the more explicit `else NULL`.